source/features/highest-rated-comment.tsx

```diff
@@ -0,0 +1,82 @@
+import './highest-rated-comment.css';
+import React from 'dom-chef';
+import select from 'select-dom';
+import features from '../libs/features';
+import * as icons from '../libs/icons';
+
+type Props = {
+	id: string;
+	text: string;
+	avatar: HTMLImageElement;
+}
+
+const element = ({id, text, avatar}: Props): Node => (
+	<div className="timeline-comment-wrapper">
+		{avatar}
+
+		<a href={`#${id}`} className="no-underline">
+			<div className="bg-white details-reset rounded-1 rgh-highest-rated-comment">
+				<div className="bg-gray border-bottom-0 px-2 py-0">
+					<div className="d-flex flex-items-center">
```

Comment:
Can any of these elements be merged or dropped? At the very least, the initial `a` should have the classes `bg-white details-reset rounded-1 rgh-highest-rated-comment` and that div can be dropped already.